airbyte-integrations/connectors/source-mssql/metadata.yaml

```diff
@@ -18,20 +18,18 @@ data:
   name: Microsoft SQL Server (MSSQL)
   registries:
     cloud:
-      # CI pipeline is broken for mssql
-      dockerImageTag: 3.6.1
       enabled: true
     oss:
-      # CI pipeline is broken for mssql
-      dockerImageTag: 3.6.1
       enabled: true
-  releaseStage: alpha
+  releaseStage: generally_available
   supportLevel: community
```

Comment:
this is very confusing that we are highlighting it as "certified' but keeping it community

Reply:
thanks @katmarkham . I understand that
But we're on our way to certified.
If GA works well and no major issues found we will officially certify it.